docs/guides/index.md

```diff
@@ -13,32 +13,31 @@ In your application, you must first configure your uPort object with an identifi
  
 ```javascript
 import { Credentials } from 'uport-credentials'
- 
+import { Resolver } from 'did-resolver'
+import { getResolver } from 'ethr-did-resolver'
+
+const providerConfig = { rpcUrl: 'https://mainnet.infura.io/<YOUR INFURA PROJECT ID>' }
+const resolver = new Resolver(getResolver(providerConfig))
+
 // For ethereum based addresses (ethr-did)
 const credentials = new Credentials({
   appName: 'App Name',
```

Comment:
I think that the `appName` and `privateKey` fields are no longer available and should be replaced with `signer: SimpleSigner(privateKey)`

Reply:
The options are still available.
Also, at this point there's not much difference between sending the private key directly or using the `SimpleSigner` which also needs direct access to the key.